chain-vote/src/committee.rs

```diff
@@ -135,6 +135,10 @@ impl MemberPublicKey {
     pub fn to_bytes(&self) -> Vec<u8> {
         self.0.to_bytes()
     }
+
+    pub fn from_public_key(pk: PublicKey) -> Self {
```

Comment:
This could be just a `From<PublicKey>` impl, so that you could do `pk.into()`.
Why is `MemberPublicKey` a newtype, anyway?